pydicom/valuerep.py

```diff
@@ -69,76 +92,75 @@ def __new__(cls, val):
             A string conformant to the DA definition in the DICOM Standard,
             Part 5, :dcm:`Table 6.2-1<part05/sect_6.2.html#table_6.2-1>`.
         """
+        if val is None:
+            return None
+
         if isinstance(val, str):
+            if val == '':
+                return None  # empty date
```

Comment:
It's a bit inconsistent for whether `''` returns `''` or `None`. DA, DT and TM will return `None`, DS and IS will return `''`